include/SKSE/Impl/PCH.h

```diff
@@ -474,7 +474,7 @@ namespace SKSE
 
 		[[noreturn]] inline void report_and_fail(std::string_view a_msg, std::source_location a_loc = std::source_location::current())
 		{
-			const auto body = [&]() {
+			const auto body = [&]() -> std::string {
```

Comment:
This is unecessary.

Reply:
this one fixes an IDE error wherein the it thinks the function returns void

Reply:
that's an issue with your ide, the compiler deduces the return type just fine.